health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccessWrapper.js

```
@@ -14,8 +14,12 @@ export const useUserAccessContext = () => {
 const UserAccessWrapper = ({ onSelect, props: customProps }) => {
   const { t } = useTranslation();
   const { state } = useMyContext();
+  const [data,setData]=useState(null);
   // vertical stepper array role code fetch and sorted based on orderNumber
   const rolesArray = state?.rolesForMicroplan?.sort((a, b) => a.orderNumber - b.orderNumber).map((item) => item.roleCode);
+  let mpRolesArray=rolesArray.map((item)=>t(`MP_ROLE_${item}`));
```

Biome

[error] This let declares a variable that is only assigned once.

'mpRolesArray' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)